public/assets/js/scripts.js

```diff
@@ -285,4 +294,131 @@ $(document).ready(function () {
+                                    <h6 class="mb-0">
+                                      ${output[i].title}
+                                      <span class="recommendation-card-icon">&nbsp; ⌄ </span>
+                                    </h6>
+                                  </a>
+                                </div>
+                                <div id="collapse${i}" class="collapse" aria-labelledby="heading${i}" data-parent="#accordion">
+                                  <p class="heading">Requirements</p>
+                                  <div class="card-body">${output[i].message}</div>
+                                  <p class="target">Target:</p>
+                                  <h4 class="price">NGN ${formatToCurrency(avgPropertyCost)}</h4>
+                                </div>
+                              </div>`;
+      }
+      recommendation = `<div id="accordion">
+                          ${recommendationBody}
+                        </div>`;
+    } else {
+      for (let i = 0; i < output.length; i++) {
+        if (i == 0) {
+          recommendationNav += `<a class="nav-item nav-link active" id="nav-tab-${i}" data-toggle="tab" href="#nav-${i}" role="tab" aria-controls="nav-${i}" aria-selected="true">
+                                  ${output[i].title}
+                                </a>`;
+        recommendationBody += `<div class="tab-pane fade active show" id="nav-${i}" role="tabpanel" aria-labelledby="nav-tab-${i}">
+                                <p class="heading">Requirements</p>
+                                <p class="body">${output[i].message}</p>
+                              </div>`
+        } else {
+          recommendationNav += `<a class="nav-item nav-link" id="nav-tab-${i}" data-toggle="tab" href="#nav-${i}" role="tab" aria-controls="nav-${i}" aria-selected="true">
+                                  ${output[i].title}
+                                </a>`;
+          recommendationBody += `<div class="tab-pane fade" id="nav-${i}" role="tabpanel" aria-labelledby="nav-tab-${i}">
+                                  <p class="heading">Requirements</p>                        
+                                  <p class="body">${output[i].message}</p>
+                                </div>` 
+        }
+      }
+      recommendation = `<nav>
+                          <div class="nav nav-tabs" id="nav-tab" role="tablist">
+                            ${recommendationNav}
+                          </div>
+                        </nav>
+                        <div class="tab-content" id="nav-tabContent">
+                          ${recommendationBody}
+  
+                          <p class="target">Target:</p>
+                          <h4 class="price">NGN ${formatToCurrency(avgPropertyCost)}</h4>
+                        </div>`;
+    }
+    
+
+    $('.search-ready-awesome-recommendation').html(recommendation);
+    $(".search-ready-awesome-div").show(1000);
```

Comment:
```suggestion
    $(".search-ready-awesome-div").slideDown();
```
I think slideDown will be a better effect than show 😉